src/fdb_kvdb.c

```diff
@@ -837,10 +837,10 @@ static void sector_iterator(fdb_kvdb_t db, kv_sec_info_t sector, fdb_sector_stor
     sec_addr = db_oldest_addr(db);
     do {
         traversed_len += db_sec_size(db);
-        read_sector_info(db, sec_addr, sector, false);
+        if (FDB_NO_ERR != read_sector_info(db, sec_addr, sector, false))return;
```

Comment:
```suggestion
        if (FDB_NO_ERR != read_sector_info(db, sec_addr, sector, false))
            return;
```
感谢 PR ，把代码的格式修正一下就可以了